### PR TITLE
[Feature] middlewares to ignore 'silent' mutation && FSB support to standards

### PR DESCRIPTION
**Pull request by blake-newman**
- Added support for 'quiet' flag to middlewares
- Fixed FSB standards https://github.com/acdlite/flux-standard-action
- Updated tests
- Updated docs
- Updated API doc to include 'object style mutation'

Note: Modifies FSB support which could mean breaking changes for users.

### CODE REVIEW

**Comment by blake-newman** [comment]
WIP DON'T MERGE, found an issue and need to add tests

File: docs/en/mutations.md [hunk 1]
```diff
@@ -67,6 +67,24 @@ mutations: {
 }
 ```
 
+### Quite Dispatch
```
Comment (azamat-sharapov) [inline docs/en/mutations.md]: typo here (and below, in this file)

Comment (azamat-sharapov) [inline docs/en/mutations.md]: btw, maybe rename it into something like `silent`? because even English man did a mistake :D

Comment (blake-newman) [inline docs/en/mutations.md]: Haha a dyslexic english man, yeah silent is nicer. I'll ammend.

**Comment by yyx990803** [comment]
What are the use cases for silent mutations? I think having silent mutations breaks that contract that all state changes are tracked by the devtool, not sure if that's a good idea unless there's a specifically convincing use case.

**Comment by blake-newman** [comment]
My use case is that I have a progress bar that by the action will dispatch a mutation every so often to update the percent, this floods the devtools and logging. 

```
export function start(store, options = {}) {
  clearInterval(timer);
  store.dispatch(START_PROGRESS_BAR, options.state);

  const cut = 10000 / Math.floor(options.duration || DEFAULT_DURATION);
  timer = setInterval(() => {
    store.dispatch({
      type: INCREASE_PROGRESS_BAR,
      silent: true,
      payload: {
        amount: cut * Math.random(),
      },
    });
    if (store.state.progressBar.percent > 95) {
      store.dispatch(FINISH_PROGRESS_BAR, options.finishState);
    }
  }, options.updateInterval || DEFAULT_UPDATE_INTERVAL);
}
```

For this i really don't care about those mutations, only the directly called actions. 

I'm assured this is not typically best practise to mutate silently. However, in some cases where there is many mutations firing due to an interval, and these mutations generally don't need to be tracked.

**Pull request merged**